act-rules-format/act-rules-format.bs

```diff
@@ -239,26 +238,12 @@ A rule was designed to test a minimum accessibility requirement and there exists
 <ul>
 <li>Conformance Requirement: Success Criterion 1.4.3 Contrast (Minimum)</li>
 <li>Secondary Requirement: Success Criterion 1.4.6 Contrast (Enhanced)</li>
-  <ul><li>Background Section: Success Criterion 1.4.6 is mapped as a Secondary requirement because the SC may be [=not satisfied=]when the rule’s outcomes are `passed`. </li></ul>
+  <ul><li>Explanation: This success criterion is **more strict** than this rule. This is because this criterion has a higher minimum contrast. Some of the passed examples do not satisfy this success criterion.</li></ul>
 </ul>
 </blockquote>
 </aside>
 
-**Scenario 2**: the rule's failed outcomes require further testing for the accessibility requirement
-
-A rule was designed to test a specific type of solution for an accessibility requirement, but the rule does not cover all solutions that can be used to meet the requirement. In this scenario, the rule’s failed outcomes cannot determine that an accessibility requirement is not satisfied because further testing is needed. The rule’s passed outcomes can determine that an accessibility requirement is satisfied. The accessibility requirement is a Secondary requirement.</li>
-
-  <aside class=example>
-    <header>Example: a rule that tests if a focusable element has no keyboard trap via standard navigation</header>
```

Comment:
Should we keep this as an (second) example of the **less strict** case?

Reply:
yes, will add